docs/develop/go/observability.mdx

```diff
@@ -156,6 +156,17 @@ It also uses Jaeger for tracing.
 
 Yes. Multiple context propagators help to structure code with each propagator having its own scope of responsibility.
 
+### Context Propagation Over Nexus Operation Calls
```

Comment:
@bergundy @Quinn-With-Two-Ns can we add a similar section to the Java SDK guide?

Reply:
Java doesn't have a comprehensive context propagation section and I didn't want to take on adding all of this extra non-Nexus related stuff for now.

I think we should add it though, just don't think Nexus should be treated specially here.

I would open an issue and ask the SDK team to prioritize since it's not a trivial amount of work.